packages/amplify-graphql-model-transformer/src/resolvers/rds/resolver.ts

```diff
@@ -1,3 +1,5 @@
+// TODO: Split this file into seperated query, mutations and subscriptions files.
```

Comment:
Is there a task tracking this todo?

Reply:
Currently the RDSModelVTLGenerator is just a place holder, the correctness of these RDS specific VTLs aren't tested. This should be handled in the PR for Model directive.